google-cloud-examples/pom.xml

```diff
@@ -76,6 +76,7 @@
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-storage</artifactId>
+      <version>1.105.0</version>
```

Comment:
when the libraries-bom gets updated, we can remove this version number

Reply:
Is it okay to move forward with it for now?

Reply:
yep fine with me!